development/kops/install.sh

```diff
@@ -0,0 +1,6 @@
+set -eo pipefail
+curl -LO https://github.com/kubernetes/kops/releases/download/v1.18.2/kops-linux-amd64
+chmod +x kops-linux-amd64
+mv kops-linux-amd64 /usr/local/bin/kops
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
```

Comment:
Remove this file this is covered by download_kops.sh

Reply:
We still need `kubectl` right? Can we just install it in-situ in the script where we use it? Or include in the documentation that users must have kubectl installed before running the kops scripts?

Also I think we should use the build of kubectl that EKS-D vends - `https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/v1.18.9/bin/linux/amd64/kubectl`. Templatizing the kubectl version in the URL is a good idea too.

Reply:
Yeh, good call on kubectl